sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/HashAggregateExec.scala

```diff
@@ -54,8 +54,7 @@ case class HashAggregateExec(
     resultExpressions: Seq[NamedExpression],
     child: SparkPlan)
```

Comment:
nit: could you remove the unused import `import org.apache.spark.sql.catalyst.plans.physical._`?

Reply:
@maropu - sure, updated.